packages/uploadthing/src/internal/handler.ts

```diff
@@ -381,7 +381,7 @@ const handleUploadAction = Effect.gen(function* () {
     const fetchContext = yield* FetchContext;
     promise = Effect.forEach(
       presignedUrls,
-      (file) => conditionalDevServer(file.key, opts.apiKey),
+      (file) => conditionalDevServer(file.key, opts.apiKey).pipe(Effect.either),
```

Comment:
?

Reply:
makes it so the dev server doesn't kill the process if it times out in case of an upload abortion. should probably be a nicer error message though